src/components/services-view/services-view.html

```diff
@@ -35,10 +35,11 @@
         </div>
         <div ng-if="$ctrl.expandSubCatRow === $index"
              class="sub-category-row"
-             pf-card-view config="$ctrl.cardViewConfig" items="$ctrl.filteredServices">
+             pf-card-view config="$ctrl.cardViewConfig" items="$ctrl.filteredItems">
           <div class="card-container">
-            <div class="card-icon {{item.imageURL ? item.imageURL : 'pf pficon-service'}}"></div>
-            <div class="card-name">{{item.displayName ? item.displayName : item.metadata.name}}</div>
+            <div ng-if="!item.imageUrl" class="card-icon {{item.iconClass ? item.iconClass : 'pf pficon-service'}}"></div>
```

Comment:
Class `pf`? Should that be `pficon`? Not sure if `pficon-service` is really the default we want.